idb/postgres/postgres_migrations.go

```diff
@@ -2204,3 +2205,21 @@ func ClearAccountDataMigration(db *IndexerDb, state *MigrationState) error {
 
 	return nil
 }
+
+// MakeDeletedNotNullMigration makes "deleted" columns NOT NULL in tables
+// account, account_asset, asset, app, account_app.
+func MakeDeletedNotNullMigration(db *IndexerDb, state *MigrationState) error {
+	queries := []string{
+		"UPDATE account SET deleted = false WHERE deleted is NULL",
```

Comment:
Have you seen that there are cases with `deleted = NULL` after the m7 migration? If there were null values that could indicate corruption, maybe we should allow the migration to fail?